### PR TITLE
Fixed cc counting for maze and bipartite

### LINKED ISSUE #38: Fix CC manual computation for maze_search and check_bipartite
Need to explain in the comments why our manual computations for the cyclomatic complexity were different from the results obtained with lizard

### PR DESCRIPTION
**Pull request by medkamilmel**
Fixes #38.

### CODE REVIEW

File: manual_evaluations/part1_complexity/maze_search.py [hunk 1]
```diff
@@ -88,12 +91,11 @@ def maze_search(maze):
             new_x = x + dx
             new_y = y + dy
 
-            # pi += 4 -> pi = 11
-            # the condition below is evaluated as 'not (0 <= new_x && new_x < height and 0 <= new_y && new_y < width)'
+            # pi += 2 -> pi = 9
```
Comment (ArnPellesGit) [inline manual_evaluations/part1_complexity/maze_search.py:94]: Is this really only +2? You evaluated it as +4 last time

Comment (medkamilmel) [reply manual_evaluations/part1_complexity/maze_search.py:94]: Screenshot 2022-02-21 at 9 34 23 AM
Screenshot 2022-02-21 at 9 34 30 AM

After some testing, I found that a < b < c and a < b both increment the CC by the same value of 1.

Comment (ArnPellesGit) [reply manual_evaluations/part1_complexity/maze_search.py:94]: Alright cool then it LGTM

**Review by ArnPellesGit — APPROVED** [review]
LGTM

**Pull request merged**